www/index.js

```diff
@@ -739,10 +742,29 @@ function init() {
     show("#details_custom");
   }
 
-  setupSelectList($("#versions"), Object.keys(config.versions), (version) => {
+  let versions = await fetch(config.image_url + "/.versions.json", { cache: "no-cache" })
+    .then((obj) => { return obj.json(); })
+    .then((obj) => { return obj.versions_list.filter((version) => version.localeCompare("19.07.4") >= 0);});
```

Comment:
Why is this needed?

Reply:
Versions prior to 19.07.4 wouldn't offer any JSON files so rendering the selector useless.